tests/test_command.py

```diff
@@ -198,6 +199,26 @@ def test_history_indicate_current(self):
         )
 
 
+class MutationTest(TestBase):
```

Comment:
**mike bayer** (zzzeek) wrote:

it would be nice if this was in "CurrentTest" but I understand that suite doesn't have a real DB set up so this is fine for now.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/1970